src/app/cube/user-profile/components/profile-header/profile-header.component.ts

```diff
@@ -76,7 +73,7 @@ export class ProfileHeaderComponent implements OnInit {
    * Method to update user info from service after changes have been made
    */
   async updateInfo() {
-    this.user = await this.profileService.fetchUserProfile(this.user.username);
+    this.user = await this.userService.fetchUserProfile(this.user.username);
```

Comment:
I would double check these service functions were doing the same thing

Reply:
I only changed the location of the service function from profile services to user user services.